src/testFixtures/java/org/opensearch/knn/ODFERestTestCase.java

```diff
@@ -158,14 +203,69 @@ protected void wipeAllODFEIndices() throws IOException {
             }
 
             for (Map<String, Object> index : parserList) {
-                String indexName = (String) index.get("index");
+                final String indexName = (String) index.get("index");
+                if (isIndexCleanupRequired(indexName)) {
+                    wipeIndexContent(MODEL_INDEX_NAME);
```

Comment:
Why is MODEL_INDEX_NAME used here and not indexName?

Reply:
ack